### PR TITLE
add blog post with training information

### PR DESCRIPTION
**Pull request by Sparrow0hawk**

### CODE REVIEW

File: blog/_posts/2021-10-19-WinterTrainingCourses.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+author: Alex Coleman
+image:
+  thumb: computer_teaching.jpg
+
+breadcrumb: true
+---
+
+We've now confirmed some upcoming training courses which are now open for registrations! 
+
+Whether you want to learn some git, linux command line or learn more about the University's high performance computing systems we've got a course for you.
```
Comment (jhodrien) [inline blog/_posts/2021-10-19-WinterTrainingCourses.md:18]:
```suggestion
Whether you want to learn some Git, Linux command line or learn more about the University's high performance computing systems we've got a course for you.
```

File: blog/_posts/2021-10-19-WinterTrainingCourses.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title:  Winter Training Courses now open
+date:   2021-10-19
+subheadline: "Training"
+teaser: "
+ Places our now available for upcoming Research Computing training courses, sign up now!
```
Comment (jhodrien) [inline blog/_posts/2021-10-19-WinterTrainingCourses.md:7]:
```suggestion
 Places are now available for upcoming Research Computing training courses, sign up now!
```

File: blog/_posts/2021-10-19-WinterTrainingCourses.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+All our courses are hands-on virtual workshops run online via Teams with course materials available online. Find booking links on the course web pages below.
+
+## November 2021
+
+### HPC0: Introduction to Linux (Online)
+
+- Tuesday 2nd November 9am till 12.30pm
+
+This is a hands-on workshop intended as an introduction to the Linux command line and shell scripting. It is suitable for Linux, Unix and Mac OSX users. The purpose of the workshop is to give users the Linux skills to be able to handle data and files, run programs and automate workflows on a PC and on the HPC service. The content of this workshop is equally suited for people who wish to use command line Linux on a PC, in the Cloud or on a HPC platform. New and prospective HPC users are advised to take this workshop before taking the HPC1: Introduction to High Performance Computing at Leeds workshop.
```
Comment (jhodrien) [inline blog/_posts/2021-10-19-WinterTrainingCourses.md:28]: Is HPC1 Introduction to High Performance Computing at Leeds or HPC Carpentry?

**Review by jhodrien — CHANGES_REQUESTED** [review]

File: blog/_posts/2021-10-19-WinterTrainingCourses.md (hunk 3 above)
Comment (Sparrow0hawk) [reply blog/_posts/2021-10-19-WinterTrainingCourses.md:28]: Yeah, this is a brand decision Martin and I haven't quite made yet

**Pull request merged**